com.unity.render-pipelines.universal/Editor/Converter/ConverterUtils.cs

```diff
@@ -0,0 +1,135 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace UnityEditor.Rendering.Universal.Converters
+{
+    public class ConverterUtils
```

Comment:
This should be `static`
```suggestion
    public static class ConverterUtils
```